mmpose/datasets/datasets/top_down/topdown_coco_dataset.py

```diff
@@ -435,3 +448,13 @@ def _do_python_keypoint_eval(self, res_file):
         info_str = list(zip(stats_names, coco_eval.stats))
 
         return info_str
+
+    def _drop_repeated(self, kpts, key='bbox_id'):
+        for img_id, bboxes in kpts.items():
+            num = len(bboxes)
+            kpts[img_id] = sorted(kpts[img_id], key=lambda x: x[key])
```

Comment:
Is the order of kpts essential for this function?

Reply:
If so, then the function name could be `_sort_and_unique_bboxes`

If not, you can use
```python
kpts[img_id] = list({bbox[key]: bbox for bbox in bboxes}.values())
```

Reply:
OK. I should rename the function.